lib/services/ignored_files_service.dart

```diff
@@ -47,6 +47,37 @@ class IgnoredFilesService {
     return false;
   }
 
+  Future<void> removeIgnoredMappings(List<File> files) async {
```

Comment:
I don't clearly understand when we need to call this function, let's discuss

Reply:
have added minor documentation and simplified the flow a little. Can discuss and add more documentation if it's still not clear.